Library/Formula/wordgrinder.rb

```diff
@@ -0,0 +1,15 @@
+class Wordgrinder < Formula
+  homepage "http://wordgrinder.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/wordgrinder/wordgrinder/wordgrinder-0.5.2.tar.bz2"
+  sha1 "4bcc600fbd110d9d87961feab15045b9b518b2db"
+
+  depends_on "lua"
+  depends_on "ncurses" => :build
+
+  def install
+    system "make"
+    (bin).install "bin/wordgrinder"
+    (man1).install "wordgrinder.man" => "wordgrinder.1"
+    (doc).install "README.wg"
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!